packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -365,11 +368,15 @@ class _CodeViewState extends State<CodeView>
                           executableLines: parsedScript != null
                               ? parsedScript!.executableLines
                               : <int>{},
+                          hitLines: parsedScript != null
```

Comment:
nit: put parsedScript into a local before this Row is defined so that we do not have to null check twice for executableLines and hitLines